PixiEditor/Models/Tools/BitmapOperationTool.cs

```diff
@@ -15,6 +15,7 @@ public abstract class BitmapOperationTool : Tool
         public bool ClearPreviewLayerOnEachIteration { get; set; } = true;
 
         public bool UseDefaultUndoMethod { get; set; } = true;
+        public virtual bool UsesShift { get => true; }
```

Comment:
You can simply use this

```suggestion
        public virtual bool UsesShift => true;
```

Reply:
done